app/browser/api/ledger.js

```diff
@@ -569,9 +571,14 @@ const getPublisherData = (result, scorekeeper) => {
     pinPercentage: result.pinPercentage,
     weight: result.pinPercentage
   }
-  // HACK: Protocol is sometimes blank here, so default to http:// so we can
-  // still generate publisherURL.
-  data.publisherURL = (result.protocol || 'http:') + '//' + result.publisherKey
+
+  data.publisherURL = result.publisherURL || ((result.protocol || 'http:') + '//' + result.publisherKey)
```

Comment:
should probably use `https` instead of `http` by default since that is what https://github.com/brave-intl/bat-publisher/blob/master/index.js#L100 does

Reply:
done